### PR TITLE
Update spellcheck adapter

### PR DESCRIPTION
**Pull request by cea2aj**
Update the spellcheck adapter to match the model expected by the SDK

J=SLAP-839
TEST=manual

Run a query and get the spellcheck component to pop up. Confirm that it looks correct. Also check that the spellcheck acceptance test passes

### CODE REVIEW

**Comment by creotutar** [comment]
Just for reference, acceptance tests failing with the same thing as #1292, and (with one fixed by said PR). This is blocked by a NPM update for Core. Would expect these to pass in the feature branch before merging.

**Review by creotutar — APPROVED** [review]
⛷️ ⛷️ ⛷️ LGTM

**Review by oshi97 — APPROVED** [review]

**Pull request merged**
